berserk/clients/tournaments.py

```diff
@@ -24,12 +24,64 @@ def get(self) -> models.CurrentTournaments:
     def get_tournament(self, tournament_id: str, page: int = 1):
         """Get information about a tournament.
 
-        :param tournament_id
+        :param tournament_id: tournament ID
+        :param page: the page number of the player standings to view
         :return: tournament information
         """
         path = f"/api/tournament/{tournament_id}?page={page}"
         return self._r.get(path, converter=models.Tournament.convert)
 
+    def join_arena(
+        self,
+        tournament_id: str,
+        password: str | None = None,
+        team: str | None = None,
+        should_pair_immediately: bool = False,
+    ):
+        """Join a tournament.
```

Comment:
Seems worth clarifying "arena" tournament and adding "Also unpauses if you had previously paused the tournament." from the docs.